Adyen/Util/HMACValidator.cs

```diff
@@ -95,7 +95,7 @@ public string GetDataToSign(NotificationRequestItem notificationRequestItem)
                 Convert.ToString(amount.Value),
                 amount.Currency,
                 notificationRequestItem.EventCode,
-                notificationRequestItem.Success.ToString()
+                notificationRequestItem.Success.ToString().ToLower()

```

Comment:
why do we need to lowercase this ?

Reply:
Again, I could be mistaken but I believe that the encryption done on the adyenside in done with a lowercase t or f. 
At least thats how the example on the website [docs](https://docs.adyen.com/development-resources/notifications/verify-hmac-signatures) is and how the tests in the other sdks expect the string to be.  
and this is where I got a false negative when I was implementing this code. changing the boolean toString with toLower fixed it. Because this wil result in a "true" string instead of a "True" string.

Reply:
@robertftenbosch thanks your right!

Reply:
Also validated on my side, for the signatures to match I had to ToLower() on the Success parameter, I ended up copy pasting the implementation to my project (to pull the fields and concat them, the function is not available on 3.2.3) and came across the same issue.